options.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+let ignoreRegExpInput;
+let visibilityInput;
+
+document.addEventListener('DOMContentLoaded', () => {
+	visibilityInput = document.querySelector('#visibilityOption');
+	ignoreRegExpInput = document.querySelector('#ignoreRegExp');
+
+	// Don't allow delimiters in RegExp string
```

Comment:
well, the delimiters need to be removed from the string before the string is converted to a `RegExp`, otherwise the delimiters sneak into the regular expression:

```js
// with delimiters in string
new RegExp('/foo/');
//=> /\/foo\// 

// without delimiters in string
new RegExp('foo')
//=> /foo/
```
naturally, we could choose to remove the delimiters in the string right before it is converted to a regex --- which could clean the code up a little bit, but since we need to remove them anyway, I figured we would wire it into the UI to prompt the user that they don't have to think about delimiters or flags. Since it seems that this extension only needs very simple pattern matching using the `|` (or) operator --- scrubbing the delimiters offers the users a visual reminder to keep their regexp focused.